testing/chain.go

```diff
@@ -36,6 +36,15 @@ import (
 	"github.com/cosmos/ibc-go/v3/testing/simapp"
 )
 
+const (
+	MAX_ACCOUNTS = 10
```

Comment:
I suggest this should be a variable so that it could be overridden in the `init()` function run on package initialisation by any module under test, for example similarly to how to we use `DefaultTestingAppInit` in the [interchain-accounts demo repo](https://github.com/cosmos/interchain-accounts/blob/master/x/inter-tx/keeper/keeper_test.go#L37):

```
var (
    MaxAccounts = 10
)

Reply:
done. great idea